guides/_subsections/configure-graphical-notifications.md

```diff
@@ -1,25 +1,25 @@
 ## [Configure Graphical Notifications](#configure-graphical-notifications)
-Actions such as starting/stopping/archiving recordings and state changes such as recordings stopping after a fixed duration
+Actions such as `Starting/Stopping/Archiving` `Recordings` and state changes such as `Recordings Stopping` after a fixed duration
```

Comment:
I don't think "starting/stopping/archiving" should be capitalized nor in a code block. Later on in the same sentence, "stopping" also shouldn't be capitalized not in a code block.